packages/litexa/src/command-line/api/smapi.coffee

```diff
@@ -96,8 +96,14 @@ module.exports = {
           logger.verbose "SMAPI statusCode #{response.statusCode}"
           data.stdout = JSON.stringify response.body, null, 2
 
-      if data.stderr and data.stderr.indexOf('ETag') < 0
-        throw data.stderr
+      if data.stderr
+        if data.stderr.indexOf('ETag') >= 0
+          # some v1 commands returned an ETag, don't need it
+        else if data.stderr.indexOf('This is an asynchronous operation') >= 0
```

Comment:
suggestion - so that we can hopefully proof this for future iterations of this string, can you add an ignore case and limit the phrase checking to "asynchronous operation"?